### PR TITLE
Create new file/folder labels

### PR DESCRIPTION
**Pull request by Pytal**
- For https://github.com/nextcloud/server/issues/36716

## Summary

Add labels for the submit buttons
image

## TODO

- [x] Files view
- [x] Public files editable view

## Related

- [x] https://github.com/nextcloud/text/pull/4083

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Comment by juliushaertl** [comment]
Failure unrelated

**Comment by artonge** [comment]
This breaks master when no `$actionLabel` is set. Should we have a default value for it?

File: lib/public/Files/Template/TemplateFileCreator.php [hunk 1]
```diff
@@ -103,6 +107,21 @@ public function getOrder(): int {
...
+		$this->actionLabel = $actionLabel;
+		return $this;
+	}
+
+	/**
+	 * @since 27.0.0
+	 */
+	public function getActionLabel(): string {
+		return $this->actionLabel;
+	}
```
Comment (nickvergessen) [inline lib/public/Files/Template/TemplateFileCreator.php:123]: Please document in https://github.com/nextcloud/server/issues/37039

**Pull request merged**